README.md

```diff
@@ -98,6 +98,64 @@ Replace ``address`` with the actual adress and replace myhomebrew with the name
 
 If no result is returned, make sure to build with the ``-g`` or``-g3`` option to make sure psp-addr2line knowns the function names and locations.
 
+## Debugging with GDB
+
+Sometimes debugging with the built in instruction level debugger is not enough, this is where
+PSPLINK's GDB Server can come in handy.
+
+Let's get started:
+
+### Preparation
+
+Prepare a separate terminal for `usbhostfs_pc`, `pspsh` and `psp-gdb`. All of them are needed to point the directory location to your `.prx` and `.elf` files located or the file you need to debug.
+
+### 1. usbhostfs_pc
+
+Run `usbhostfs_pc` on your terminal dedicated for `usbhostfs_pc` and you will see the `waiting for device...` status and then start the PSPLINK app on your PSP and connect the USB cable, you will see the `connected to device` status, it means success. do not close this terminal after that.
```

Comment:
```suggestion
Run `usbhostfs_pc` on your terminal dedicated for `usbhostfs_pc` and you will see the `waiting for device...` status. Now start the PSPLINK app on your PSP and connect the USB cable. You should see the `connected to device` status in the terminal, which means success. Do not close this terminal after that.
```

Reply:
Not sure if this suggestion is the best wording, but the long sentence would be nice to break up.